msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -199,6 +202,57 @@ public AcquireTokenResult acquireTokenSilent(AcquireTokenSilentOperationParamete
         }
     }
 
+    /**
+     * Get MSAL PublicClientApplication mode from Broker.
+     * */
+    public void getBrokerAccountMode(final Context appContext,
```

Comment:
Same question here - should this be on the interface?

Reply:
Same answer as above.

Reply:
This one is in another class.